client/src/components/review-items/review-items.tsx

```diff
@@ -108,87 +108,85 @@ export default function ReviewItems({ items, isLoading }: Props) {
+                        {item.mainText}{" "}
+                        <span className="text-gray-400">(#{item.id})</span>
+                      </h2>
+                    </SheetTitle>
+
+                    <div className="my-4">
+                      <p className="leading-7 [&:not(:first-child)]:mt-6">
+                        {parsingSubtext(item.subText)}
+                      </p>
+                      <p className="leading-7 [&:not(:first-child)]:mt-6">
+                        Next review at{" "}
+                        <Badge style={randomColor()}>{nextReviewDate}</Badge>
+                      </p>
+                    </div>
+                  </header>
+
+                  <section>
+                    <div className="flex flex-col items-center my-4">
+                      <h4 className="text-xl font-semibold tracking-tight text-black scroll-m-20">
+                        Review Dates
+                      </h4>
+
+                      <div className="text-center">
+                        <Calendar selected={reviewDates} />
                       </div>
-                    </header>
-
-                    <section>
-                      <div className="flex flex-col items-center my-4">
-                        <h4 className="text-xl font-semibold tracking-tight text-black scroll-m-20">
-                          Review Dates
-                        </h4>
-
-                        <div className="text-center">
-                          <Calendar selected={reviewDates} />
-                        </div>
-                      </div>
-                    </section>
-
-                    <footer>
-                      <div className="flex flex-col w-full gap-y-4">
-                        <Button onClick={() => markAsMemorized(item.id)}>
-                          Memorized
-                        </Button>
-                        <Button
-                          onClick={() => remindMeAgainLater(item.id)}
-                          variant="secondary"
-                        >
-                          Remind me again later
-                        </Button>
-                        <Button
-                          onClick={() => deleteItem(item.id)}
-                          variant="destructive"
-                        >
-                          Delete
-                        </Button>
-                      </div>
-                    </footer>
+                    </div>
                   </section>
-                </SheetContent>
-              </Sheet>
-            );
-          })
-        )}
+
+                  <footer>
+                    <div className="flex flex-col w-full gap-y-4">
+                      <Button onClick={() => markAsMemorized(item.id)}>
+                        Memorized
+                      </Button>
+                      <Button
+                        onClick={() => remindMeAgainLater(item.id)}
```

Comment:
It's just trivial. But what about using `remindMeLater` instead of `remindMeAgainLater`?